components/contract-tabs/permissions/ContractPermission.tsx

```diff
@@ -47,7 +47,7 @@ export const ContractPermission: React.FC<IContractPermission> = ({
           <Flex>
             <Stack spacing={1} flexGrow={1}>
               <Heading size="heading.sm" textTransform="capitalize">
-                {role}
```

Comment:
this means we'd rename it for all contracts, is that intended? Or did we mean to only do it for drops?

Reply:
I think it's okay renaming for all contracts, if you believe it's better to do it only for drops, I can check contractType here and do it only then

Reply:
I just thought about it again, and NFT Collection and Edition have signature based minting, which allows people to mint without the minter role, so it can also get confusing on those.

Reply:
ok sounds good, if it becomes confusing we'll need to react but :shipit: for now